workers/tests/unittests/test_api.py

```diff
@@ -39,3 +39,25 @@ def test_remove_svg_from_properties(self):
         self.assertNotIn("result.svg", list(feature["properties"].keys()))
         for element in feature_collection["features"]:
             self.assertNotIn("result.svg", list(element["properties"].keys()))
+
+    def test_remove_html_from_properties(self):
```

Comment:
I would split the test into two or more test functions.
At least one for a single feature and one for a feature collection.

Reply:
Adressed in #280